libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1622,6 +1622,18 @@ void GCS_MAVLINK::send_accelcal_vehicle_position(uint32_t position)
     }
 }
 
+void GCS_MAVLINK::reboot_zero_rc_outputs()
+{
+    // Send an invalid signal to the motors to prevent spinning due to neutral (1500) pwm pulse being cut short
+    // For that matter, send an invalid signal to all channels to prevent undesired/unexpected behavior
+    hal.rcout->cork();
+    for (int i=0; i<NUM_RC_CHANNELS; i++) {
+        // Set to 1 because 0 is interpreted as flag to ignore update
+        hal.rcout->write(i, 1);
+    }
+    hal.rcout->push();
```

Comment:
SRV_Channels::push() and RCOutput::push() are not the same thing.

Reply:
Fixed.